user-workspace/src/App.tsx

```
@@ -22,77 +23,57 @@ const config = {
+    if (!inputText) {
       Snackbar.show({
-        text: 'Verification URL is required',
+        text: 'Input is required',
         duration: Snackbar.LENGTH_LONG,
       });
       return;
     }
 
+    switch (verificationMethod) {
+      case 'provider':
+        console.assert(config.REACT_APP_RECLAIM_APP_ID, 'RECLAIM_APP_ID is not set');
```

ESLint

[error] Replace `config.REACT_APP_RECLAIM_APP_ID,·'RECLAIM_APP_ID·is·not·set'` with `⏎··········config.REACT_APP_RECLAIM_APP_ID,⏎··········'RECLAIM_APP_ID·is·not·set'⏎········`

(prettier/prettier)